client-src/elements/form-field-enums.js

```diff
@@ -204,7 +204,7 @@ export const INTENT_STAGES = {
   INTENT_IMPLEMENT: [1, 'Start prototyping'], // Start prototyping
   INTENT_EXPERIMENT: [2, 'Dev trials'], // Dev trials
   INTENT_IMPLEMENT_SHIP: [4, 'Evaluate readiness to ship'], // Eval readiness to ship
-  INTENT_EXTEND_TRIAL: [3, 'Origin Trial'], // Origin trials
+  INTENT_ORIGIN_TRIAL: [3, 'Origin Trial'], // Origin trials
```

Comment:
Do you also need the new enum value here?

Reply:
Yes! added.